packages/components-react/projects/nextjs/components/Select.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import { routes } from '../routes';
+import { useState } from 'react';
+import { useRouter, usePathname } from 'next/navigation';
+import { componentsReady } from '@porsche-design-system/components-react';
+
+export const Select = (): JSX.Element => {
+  const router = useRouter();
+  const pathname = usePathname();
+  const [selected, setSelected] = useState(pathname);
+  return (
+    <select
+      value={selected as string}
+      onChange={(e) => {
+        const { value } = e.target;
+        setSelected(value);
+        router.push(value);
+      }}
+    >
+      <option disabled value="">
+        Select a page
+      </option>
+      {routes.map((route) => (
+        <option key={route.path} value={route.path} children={route.name} />
+      ))}
+    </select>
+  );
+};
+
+if (typeof window !== 'undefined') {
+  (window as any).componentsReady = componentsReady; // for vrt
```

Comment:
Seems like the wrong place to have something this essential in a random component.